### ISSUE TITLE
new nudge notice occasionally doesn't show up on youtube

### ISSUE THREAD

**Issue by gkovacs** [habitlab/habitlab#424]
issue seems restricted to youtube and not any other site. when the issue occurs, buttons also lose their padding. perhaps something with youtube's built in version of polymer is conflicting with our copy? occurs only very occasionally though, suggesting a race condition or so

**Comment by gkovacs** [comment]
notice also doesn't show up completely reliably on reddit either, reddit definitely doesn't use polymer so something else is the cause in the case of reddit though. this one seems to happen only with the generic/show_timer_banner intervention

```
VM21:5 Uncaught TypeError: Cannot read property 'addEventListener' of null
    at HTMLElement.created (<anonymous>:39:125204)
    at HTMLElement.__initialize (<anonymous>:39:248886)
    at HTMLElement.createdCallback (<anonymous>:39:248611)
    at a (<anonymous>:39:120060)
    at t (<anonymous>:39:119606)
    at s (<anonymous>:39:121528)
    at new <anonymous> (<anonymous>:39:121649)
    at HTMLDocument.c [as createElement] (<anonymous>:39:121765)
    at Function.Polymer.IronA11yAnnouncer.requestAvailability (<anonymous>:39:125611)
    at HTMLElement.created (<anonymous>:39:150213)
```

**Comment by gkovacs** [comment]
fixed 1.0.212
